metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_resolver.py

```diff
@@ -63,10 +63,48 @@ def __init__(self, timeout, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.timeout = timeout
 
-    def request(self, method, url, **kwargs):
-        # Set the default timeout if none is provided
-        kwargs.setdefault("timeout", self.timeout)
-        return super().request(method, url, **kwargs)
+    def request(
```

Comment:
I was getting ```
src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_resolver.py:66: error: Signature of "request" incompatible with supertype "Session"  [override]
src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_resolver.py:66: note:      Superclass:
```
if I did not do this. But now there is a no-untyped defs. The stuff is internal to the parent. Not sure what should be done here